api/src/templates/project_types.html

```diff
@@ -2,20 +2,21 @@
 <html lang="en" dir="ltr">
  <head>
   <meta charset="utf-8">
-  <title></title>
+  <link rel="stylesheet" href="../static/styles/main.css">
+  <title>Project Types</title>
  </head>
  <body>
   <nav>
-   <a href="/forms/projects">All projects</a>
+   <button href="/forms/projects">Back to projects</button>
```

Comment:
Should this button be wrapped in an anchor tag? The links between pages aren't working anymore.

Reply:
Yep, you're right! Thanks. I can make sure the links are all working when you let me know what the endpoint URLs should be like I explained in the below comment.